tests/test_models/test_nn/deepar/test_loss.py

```diff
@@ -0,0 +1,127 @@
+import pytest
+import torch
+from pytorch_lightning import seed_everything
+from torch.distributions import NegativeBinomial
+from torch.distributions import Normal
+
+from etna.models.nn.deepar_new.loss import GaussianLoss
+from etna.models.nn.deepar_new.loss import NegativeBinomialLoss
+
+
+@pytest.mark.parametrize(
+    "loss,distribution,loc,scale,weights",
+    [
+        (GaussianLoss(), Normal, torch.tensor([0.0]), torch.tensor([1.0]), torch.tensor([1.0])),
+        (GaussianLoss(), Normal, torch.tensor([10.0]), torch.tensor([100.0]), torch.tensor([1.0])),
+        (NegativeBinomialLoss(), NegativeBinomial, torch.tensor([2.0]), torch.tensor([2.0]), torch.tensor([1.0])),
+        (NegativeBinomialLoss(), NegativeBinomial, torch.tensor([0.6]), torch.tensor([0.1]), torch.tensor([1.0])),
+    ],
+)
+def test_sample_mean(loss, distribution, loc, scale, weights):
+    seed_everything(0)
+    mean = loss.sample(loc=loc, scale=scale, weights=weights, theoretical_mean=True)
+    scaled_params = loss.scale_params(loc, scale, weights)
+    seed_everything(0)
+    expected_mean = distribution(scaled_params[0], scaled_params[1]).mean
+    torch.testing.assert_close(mean, expected_mean, atol=1e-10, rtol=1e-10)
+
+
+@pytest.mark.parametrize(
+    "loss,distribution,loc,scale,weights",
+    [
+        (GaussianLoss(), Normal, torch.tensor([0.0]), torch.tensor([1.0]), torch.tensor([1.0])),
+        (GaussianLoss(), Normal, torch.tensor([1.0]), torch.tensor([2.0]), torch.tensor([1.0])),
+        (NegativeBinomialLoss(), NegativeBinomial, torch.tensor([2.0]), torch.tensor([2.0]), torch.tensor([1.0])),
+        (NegativeBinomialLoss(), NegativeBinomial, torch.tensor([10.0]), torch.tensor([0.2]), torch.tensor([1.0])),
+    ],
+)
+def test_sample_random(loss, distribution, loc, scale, weights, n_samples=200):
```

Comment:
The idea was to compare the mean of the samples with theoretical mean of the distribution, you just reimplement the method